### PR TITLE
Fixed #401 - Random places in the addon are in another language when using the English version

### LINKED ISSUE #401: Random places in the addon are in another language when using the English version
For some reason, in random places the addon switches to another language instead of English. These screenshots show the problem areas.

WoWScrnShot_030219_011238
WoWScrnShot_030219_011318
WoWScrnShot_030219_011309

### PR DESCRIPTION
**Pull request by Dillon-Brown**
## Description
Fixed a few issues with the enUS translation containing non-english strings. See the original issue below for UI screenshots to see where these appear. 

## Motivation and Context
Issue reference: #401

### CODE REVIEW

File: Locales/enUS.lua [hunk 1]
```diff
@@ -102,9 +102,9 @@ AtrL["Show auction house prices in mailbox tooltips"] = ""
 AtrL["Undercutting"] = ""
 AtrL["Selling"] = ""
 AtrL["Configure how you typically like to sell the items listed below."] = ""
-AtrL["Clears"] = "Pulizia"
+AtrL["Clears"] = ""
```
Comment (Pretzellent) [inline Locales/enUS.lua]: I think this should be "Cleanup" or "Clear Data"
not an empty string

File: Locales/enUS.lua [hunk 2]
```diff
@@ -102,9 +102,9 @@ AtrL["Show auction house prices in mailbox tooltips"] = ""
 AtrL["Undercutting"] = ""
 AtrL["Selling"] = ""
 AtrL["Configure how you typically like to sell the items listed below."] = ""
-AtrL["Clears"] = "Pulizia"
+AtrL["Clears"] = ""
 AtrL["The features below will help you clear out data that Auctionator stores.  Typically you would only need this in the event that that data gets corrupted."] = ""
-AtrL["Read the FAQ at"] = "Leggi le FAQ su"
```
Comment (Pretzellent) [inline Locales/enUS.lua]: "Read the FAQ on"

Comment (Dillon-Brown) [reply Locales/enUS.lua]: Done

File: Locales/enUS.lua (hunk 1 above)
Comment (Dillon-Brown) [reply Locales/enUS.lua]: Done

**Comment by jrob8577** [comment]
Thanks for your contributions!!

**Pull request merged**